base/timing.jl

```diff
@@ -353,9 +353,11 @@ macro timev(msg, ex)
         Experimental.@force_compile
         local stats = gc_num()
         local elapsedtime = time_ns()
+        cumulative_compile_timing(true)
         local compile_elapsedtimes = cumulative_compile_time_ns()
         local val = @__tryfinally($(esc(ex)),
             (elapsedtime = time_ns() - elapsedtime;
+            cumulative_compile_timing(false)
```

Comment:
```suggestion
            cumulative_compile_timing(false);
```
you're missing a semicolon here for the macro expression

Reply:
I went ahead and committed it for you, to restart the tests. hope that's okay

Reply:
Yep thanks